### PR TITLE
Improvement: Bundle update

### PR DESCRIPTION
**Pull request by snova-luiss**

### CODE REVIEW

File: bundle_jump_start/README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ The config.yaml file is crucial for customizing the behavior of the CoE LLM Rout
       "temperature": 0.0
       "do_sample": False
       "max_tokens_to_generate": 1200
-      "coe": True #set as true if using Sambastudio CoE endpoint
+      "bundle": True #set as true if using Sambastudio CoE endpoint
```
Comment (snova-rodrigom) [inline bundle_jump_start/README.md:99]: Please change this CoE reference and check if there are others.

**Review by snova-jorgep — CHANGES_REQUESTED** [review]
fix typo (minor)

**Review by snova-rodrigom — CHANGES_REQUESTED** [review]
LGTM, dropped a small comment and could you confirm if the kits impacted have been tested?

**Review by snova-jorgep — APPROVED** [review]

**Review by snova-rodrigom — APPROVED** [review]
LGTM now, thanks @snova-luiss

**Pull request merged**